packages/statcord/README.md

```diff
@@ -0,0 +1,275 @@
+
+More information about Statcord can be found on its [website](https://statcord.com).
+
+## Features
+
+-   Fully ready for TypeScript!
+-   Includes ESM ready entrypoint
+-   Easy to use
+
+## Installation
+
+`@sapphire/plugin-statcord` depends on the following packages. Be sure to install these along with this package!
+
+-   [`@sapphire/framework`](https://www.npmjs.com/package/@sapphire/framework)
+
+You can use the following command to install this package, or replace `npm install` with your package manager of choice.
+
+```sh
+npm install @sapphire/plugin-statcord @sapphire/framework
+```
+
+---
+
+## Usage
+
+### JavaScript
+
+In your main or setup file, register the plugin:
+
+```javascript
+require('@sapphire/plugin-statcord/register');
+```
+
+It is important to add an API key provided by Statcord.
+
+```javascript
+require('@sapphire/plugin-statcord/register');
+
+const client = new SapphireClient({
+	/* your bot options */
+	statcord: {
+		key: 'YOUR_AWESOME_API_KEY', // (Required) Statcord API key.
+		autopost: false, // (Optional) Allows automatic posting of statistics.
+		baseUrl: 'https://api.statcord.com/v3', // (Optional) Change the base URL of the Statcord API.
+		debug: false, // (Optional) Show debug messages.
+		sharding: false // (Optional) Activate the sharding mode, it is important to read the notes below.
+	}
+});
+
+async function main() {
+	await client.login();
+}
+
+void main();
+```
+
+### TypeScript
+
+In your main or setup file, register the plugin:
+
+```typescript
+import '@sapphire/plugin-statcord/register';
+```
+
+It is important to add an API key provided by Statcord.
+
+```typescript
+import '@sapphire/plugin-statcord/register';
+
+const client = new SapphireClient({
+	/* your bot options */
+	statcord: {
+		key: 'YOUR_AWESOME_API_KEY', // (Required) Statcord API key.
+		autopost: false, // (Optional) Allows automatic posting of statistics.
+		baseUrl: 'https://api.statcord.com/v3', // (Optional) Change the base URL of the Statcord API.
+		debug: false, // (Optional) Show debug messages.
+		sharding: false // (Optional) Activate the sharding mode, it is important to read the notes below.
+	}
+});
+
+async function main() {
+	await client.login();
+}
+
+void main();
+```
+
+If you enable the `autopost` option, that's it, the plugin will collect and publish the statistics for you, you don't have to do anything else!
+
+## Bandwidth usage
+To set the bandwidth usage in each statistics post the `setBandwidthUsage()` method is available, the data sent by this method is reset in each statistics post. This is done so that the user can choose the best way to get this data.
+
+### Javascript
+```javascript
+const { container } = require('@sapphire/framework');
+
+class MyAwesomeServicePostStats {
+	public postBandwidth() {
+		const bandWitchUsage = getBandwidthUsage(); // Use your method to get this data.
```

Comment:
BandWitch? Same in [this line](https://github.com/sapphiredev/plugins/pull/231/files#diff-370e1bb2f4269f7c74bac8e5441499b12948cec2a235e4b45fde2b339f69e2d4R13D2)